modules/entity/src/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { isDevMode } from '@angular/core';
+import { IdSelector } from './models';
+
+export function selectIdValue<T>(entity: T, selectId: IdSelector<T>) {
+  const key = selectId(entity);
+
+  if (isDevMode() && key === undefined) {
+    console.warn(
```

Comment:
What about `false`?

Reply:
What do you mean by that? You would also log this when the `key` would be `null`, `false`?

Reply:
Yes. We also want to warn if the selected value is `false`, correct?

Reply:
My initial thinking was to only check on `undefined` because the id property wouldn't exist on the entity, meaning you probably did something wrong. I'm not sure how this is implemented by other devs, because if one would define an entity as `null`, this is "hes choice" and he should know what he is doing.
Therefor I'm only checking on `undefined`.

For me it would also make more sense to check if the id value is false, but as I said I'm not sure how this is used.

Reply:
Ok, that makes sense. My rationale was that we're checking for values that aren't valid ids.